pkg/sbom/cyclonedx/marshal_test.go

```diff
@@ -800,7 +830,7 @@ func TestMarshaler_Marshal(t *testing.T) {
 						},
 					},
 					{
-						BOMRef:     "pkg:gem/actionpack@7.0.0?file_path=tools%2Fproject-john%2Fspecifications%2Factionpack.gemspec",
+						BOMRef:     "pkg:gem/actionpack@7.0.0",
```

Comment:
We don't seem to expect any changes in the desired results for this test.

Reply:
addressed at https://github.com/aquasecurity/trivy/pull/5439/commits/cdd281e1e8fc988cf747b8a94e734e548c4ee310